website/source/layouts/blog-post.html.haml

```diff
@@ -0,0 +1,18 @@
+!!!html5
+%html
+  = partial "layouts/head"
+  %body.layout(class="#{"-subpage-" unless data.page.root}")
+    .layout--container
+      = partial "layouts/header"
+      .layout--content
+        %h1.blog--headline= data.page.title
+        .markdown
+          = yield.gsub(%r(\A.*<body class="livingstyleguide">(.+)</body>.*\Z)m, '\\1')
+        .social-buttons
+          .social-buttons--item
+            .fb-like(data-action="like" data-href="#{current_url}" data-layout="box_count" data-share="false")
+          .social-buttons--item
+            %a(href="Tweet" class="twitter-share-button" data-count="vertical" data-related="livingstyleguide" data-text="#{data.page.title}—LivingStyleGuide" data-url="#{current_url}") https://twitter.com/share
+          .social-buttons--item
+            .g-plusone(data-size="tall")
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))